src/instruments/src/EFB/Utils/PushbackUpdater.tsx

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2022 FlyByWire Simulations
+// SPDX-License-Identifier: GPL-3.0
+
+const SHOW_DEBUG_INFO = false;
+
+const InternalTugHeadingDegrees = 0xffffffff / 360;
+
+export class PushbackUpdater {
+    private timeout = 50;
+
+    private updateInterval = undefined;
+
+    private lastTime = 0.0;
+
+    private startTime = 0.0;
+
+    private deltaTime = 0.0;
+
+    private updateTime = 0.0;
+
+    private simOnGround = true;
+
+    private pushBackAttached = false;
+
+    private parkingBrakeEngaged = true;
+
+    private aircraftHeading = 0;
+
+    private tugCommandedSpeedFactor = 0;
+
+    private tugCommandedHeadingFactor = 0;
+
+    private computedTugHeading = 0;
+
+    constructor() {
+        console.log('PushbackUpdater initialized.');
+    }
+
+    updater(pb) {
+        pb.startTime = Date.now();
+        pb.deltaTime = pb.startTime - pb.lastTime;
+        pb.lastTime = pb.startTime;
+        SimVar.SetSimVarValue('L:A32NX_PUSHBACK_DELTA_TIME', 'Number', pb.deltaTime);
+
+        pb.simOnGround = SimVar.GetSimVarValue('SIM ON GROUND', 'bool');
+        pb.pushBackAttached = SimVar.GetSimVarValue('Pushback Attached', 'bool');
+
+        if (pb.pushBackAttached && pb.simOnGround) {
```

Comment:
be advised this is a bit problematic as noted in #efb
https://discord.com/channels/738864299392630914/747862215679279214/969858465939980348

PUSHBACK STATE is a bit buggy so have to mitigate this. Just a general comment, nothing that concerns this PR